lib/handlers/quote/handler.ts

```diff
@@ -168,6 +174,17 @@ export class QuoteHandler extends APIGLambdaHandler<
     };
   }
 
+  public getQuoteRequestSource(event: APIGatewayProxyEventHeaders): RequestSource {
+    const userAgent = event?.['x-request-source']
```

Comment:
I don't think userAgent is a good name for this as its an already existing header name, maybe something like `requestSource`? Additionally, the request source is not the userAgent